cms/templates/index.html

```diff
@@ -149,23 +122,22 @@ <h3 class="title">${_("Create a New Library")}</h3>
-                    <span class="tip">${_("The public display name for your library.")}</span>
-                    <span class="tip tip-error is-hiding"></span>
+                    <input class="new-library-name" id="new-library-name" type="text" name="new-library-name" required placeholder="${_('e.g. Computer Science Problems')}" aria-describedby="tip-new-library-name tip-error-new-library-name" />
+                    <span class="tip" id="tip-new-library-name">${_("The public display name for your library.")}</span>
+                    <span class="tip tip-error is-hiding" id="tip-error-new-library-name"></span>
                   </li>
                   <li class="field text required" id="field-organization">
                     <label for="new-library-org">${_("Organization")}</label>
-                    <input class="new-library-org" id="new-library-org" type="text" name="new-library-org" aria-required="true" placeholder="${_('e.g. UniversityX or OrganizationX')}" />
-                    <span class="tip">${_("The name of the organization sponsoring the library.")}  <strong>${_("Note: This is part of your library URL, so no spaces or special characters are allowed.")}</strong> ${_("This cannot be changed.")}</span>
-                    <span class="tip tip-error is-hiding"></span>
+                    <input class="new-library-org" id="new-library-org" type="text" name="new-library-org" required placeholder="${_('e.g. UniversityX or OrganizationX')}" aria-describedby="tip-new-library-org tip-error-new-library-org" />
+                    <span class="tip" id="tip-new-library-org">${_("The public organization name for your library.")} ${_("This cannot be changed.")}</span>
+                    <span class="tip tip-error is-hiding" id="tip-error-new-library-org"></span>
                   </li>
 
                   <li class="field text required" id="field-library-number">
-                    <label for="new-library-number">${_("Library Code/Number")}</label>
-                    ## Translators: This is an example for the "number" used to identify a library, seen when filling out the form to create a new library. This example is short for "Computer Science Problems". The example number may contain letters but must not contain spaces.
-                    <input class="new-library-number" id="new-library-number" type="text" name="new-library-number" aria-required="true" placeholder="${_('e.g. CSPROB')}" />
-                    <span class="tip">${_("The unique code that identifies this library.")} <strong>${_("Note: This is part of your library URL, so no spaces or special characters are allowed.")}</strong> ${_("This cannot be changed.")}</span>
-                    <span class="tip tip-error is-hiding"></span>
+                    <label for="new-library-number">${_("Major Version Number")}</label>
```

Comment:
@antoviaque @marcotuts per our discussion this morning, we confirmed that the final implementation should be "Library Code" (not "Major Version Number").

Please restore the previous label and tip etc. and also eliminate the "/Number" behind "Library Code" since it can be an alpha as well as numeric code. Probably clarify the "code" (vs "number") for translators as well?
Thanks! C